main.cpp

```diff
@@ -1,7 +1,29 @@
 #include <iostream>
+#include <string.h>
+
+#define MAX_LENGTH 30
+
+void to_lowercase(char* word);
```

Comment:
For stringview:

Pros:
Since it's a reference to a string
we would always keep the original
string in tact and instead make copies
that are altered to fit the specific need
(Reusability)

Cons:
We'd be passing the copies around as references
Which defeats the purpose of it


For std::string*:
Pros:
Memory is saved due to only working with one
String and not multiple copies (not so reusable though)

Cons:
If the string is altered in some improper way
It could potentially mess up the entire program
(Undefined behavior)